packages/camera/src/components/Layout/index.js

```diff
@@ -3,10 +3,13 @@ import PropTypes from 'prop-types';
 import screenfull from 'screenfull';
 import { Button, Platform, StyleSheet, useWindowDimensions, View } from 'react-native';
 
-import getOS from '../../utils/getOS';
-import useOrientation from '../../hooks/useOrientation';
 import useMobileBrowserConfig from '../../hooks/useMobileBrowserConfig';
-import PortraitOrientationBlocker from './PortraitOrientationBlocker';
+import useOrientation from '../../hooks/useOrientation';
+
+if (Platform.OS === 'web') {
+  // eslint-disable-next-line global-require
+  require('./layout.css');
+}
```

Comment:
The orientation works great on web, but I couldn't run the app on native due to the following error.

![IMG_2038](https://user-images.githubusercontent.com/50136109/164216696-657bbd04-8893-4f6b-8fe7-0e0b12b54c2f.PNG)

Reply:
Yeah of course, that was dumb of me. I need to use RN Stylesheet. ^^